packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.tsx

```diff
@@ -399,6 +420,10 @@ export class CommandBar extends BaseComponent<ICommandBarProps, ICommandBarState
 
   private _onItemClick(item: IContextualMenuItem): (ev: React.MouseEvent<HTMLButtonElement>) => void {
     return (ev: React.MouseEvent<HTMLButtonElement>): void => {
+      if (item.readOnly) {
```

Comment:
If this is added to command bar, it should also be added to the contextualmenu click handler.

Reply:
@joschect I added a comment on this prop that it's implementation is not supported by all components, like for contextualmenu and for button as micah as mentioned, then other teams can come in and add the functionality on these components as they need them. is that ok?